### PR TITLE
Guard helpful-update against keys bound to undefined commands

### PR DESCRIPTION
**Pull request by narendraj9**

### CODE REVIEW

**Comment by narendraj9** [comment]
@Wilfred Done :)

File: helpful.el [hunk 1]
```diff
@@ -747,6 +747,10 @@ For example, \"(some-func FOO &optional BAR)\"."
     (unless sym
       (user-error "No command is bound to %s"
                   (key-description key-sequence)))
+    (unless (commandp sym)
+      (user-error "%s is bound to symbol - %s, that is not a command"
```
Comment (DamienCassou) [inline helpful.el]: Why do you have a `-` in the middle of the sentence?

Comment (narendraj9) [reply helpful.el]: @DamienCassou What would you like the sentence to be?

Comment (narendraj9) [reply helpful.el]: I thought 
symbol <dash> symbol_name is better than 
symbol symbol_name

for some reason when I wrote this. Now that we are discussing this, I don't seem to be very confident about it.

Comment (DamienCassou) [reply helpful.el]: > What would you like the sentence to be?

I guess @Wilfred would be the best person to decide that, but I think your sentence without the dash and comma would be good enough.

**Comment by Wilfred** [comment]
Thank you :)

**Pull request merged**